librz/analysis/esil/esil_trace.c

```diff
@@ -18,6 +18,111 @@ static void htup_vector_free(HtUPKv *kv) {
 	rz_vector_free(kv->value);
 }
 
+RZ_API RzPVector *ht_db_get(HtPP *db, const char *key) {
+	return ht_pp_find(db, key, NULL);
+}
+
+static void htpp_vector_free(HtPPKv *kv) {
+	rz_vector_free(kv->value);
+}
+
+RZ_API void ht_db_array_add(HtPP *db, const char *key, const char *val) {
```

Comment:
does this need to be a RZ_API ? If so, you should name it in a better way because otherwise it may be confused as a generic ht function, however it is not.

Reply:
Actually it's only used for two files now. It's not really necessary to be an RZ_API, I will make them static after migrate the `string-based 'array'` to `RzPvector`

Reply:
Or If we really need something like it in other places. I will try to move it to `util` as Xvilka said.